lisa/sut_orchestrator/baremetal/ip_power.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Type
+
+import requests
+
+from lisa import schema
+from lisa.util import ContextMixin, InitializableMixin, subclasses
+from lisa.util.logger import get_logger
+
+from .schema import IPPowerSchema
+
+
+class IPPower(subclasses.BaseClassWithRunbookMixin, ContextMixin, InitializableMixin):
+    def __init__(self, runbook: IPPowerSchema) -> None:
+        super().__init__(runbook=runbook)
+        self._log = get_logger("cluster", self.__class__.__name__)
+
+    @classmethod
+    def type_schema(cls) -> Type[schema.TypedSchema]:
+        return IPPowerSchema
+
+    def on(self, port: int) -> None:
+        raise NotImplementedError()
+
+    def off(self, port: int) -> None:
+        raise NotImplementedError()
+
+
+class Ip9285(IPPower):
+    def __init__(self, runbook: IPPowerSchema) -> None:
+        super().__init__(runbook)
+        self._request_cmd = (
```

Comment:
What does the `P6` mean? If it's part of the `port`, move it to on/off, so it's clearer.